tests/common/test_cgroups.py

```diff
@@ -110,49 +98,94 @@ def test_is_active_incorrect_file(self, patch_periodic_warn, *_):
 
 
 class TestCpuCgroup(AgentTestCase):
+    @classmethod
+    def setUpClass(cls):
+        AgentTestCase.setUpClass()
+
+        original_read_file = fileutil.read_file
+
+        #
+        # Tests that need to mock the contents of /proc/stat or */cpuacct/stat can set this map from
+        # the file that needs to be mocked to the mock file (each test starts with an empty map).
+        #
+        cls.mock_read_file_map = {}
+
+        def mock_read_file(filepath, **args):
```

Comment:
Thanks for fixing this. This looks better and cleaner.

Reply:
np